ceros/ShareJS
# Feature/jenkins server

Resolves ceros/ceros-product#3735

This changes are regarding the creation of the Jenkinsfile that will be used by Jenkins. This file represent the pipeline of sharejs application based on the concept of "Pipeline as a code" where the pipeline definitions are pushed in the application repository. 

## Deploy notes:

No need to deploy.

## Testing Steps:
To test it, go to Jenkins (https://jenkins.cerosdev.com) and run 2 jobs: 
- ceros/sharejs/jenkins-server and check if the tests get executed
- sharejs-deploy and check if the version is deployed on dev8.
